addons/mass_mailing/report/mailing_trace_report_views.xml

```diff
@@ -32,6 +32,7 @@
             <field name="model">mailing.trace.report</field>
             <field name="arch" type="xml">
                 <search string="Mass Mailing Statistics">
+                    <field name="campaign"/>
```

Comment:
While you're at it, can you change this field's string attribute definition from "Mass Mail Campaign" to "Mailing Campaign" to match what we use in mailings ?